packages/ui-extensions-server-kit/src/types.ts

```diff
@@ -97,6 +97,17 @@ export interface ExtensionPoint {
 
 export type ExtensionPoints = string[] | ExtensionPoint[] | null
 
+interface CollectBuyerConsentCapabilities {
```

Comment:
Singular tense

```suggestion
interface CollectBuyerConsentCapability {
```

Reply:
Yeahhh this whole thing is wrapped in a `Capabilities` interface so my brain wasn't sure whether to fix that or keep it consistent.